test/mockcore_test.cc

```diff
@@ -80,7 +81,7 @@ class MockCoreFixture : public ::testing::Test {
     {}
 
     void SetUp() {
-        spdlog::set_level(spdlog::level::err);
+        // spdlog::set_level(spdlog::level::err);
```

Comment:
?

Reply:
trivia change, no need to log in test for now